src/CMakeLists.txt

```diff
@@ -484,17 +484,17 @@ endif()
 
 if(WITH_BLUESTORE_PMEM OR WITH_RBD_RWL)
   if(WITH_SYSTEM_PMDK)
-    set(pmem_COMPONENTS)
+    set(pmdk_COMPONENTS)
     if(WITH_BLUESTORE_PMEM)
-      list(APPEND pmem_COMPONENTS pmem)
+      list(APPEND pmdk_COMPONENTS pmem)
     endif()
     if(WITH_RBD_RWL)
-      list(APPEND pmem_COMPONENTS pmemobj)
+      list(APPEND pmdk_COMPONENTS pmemobj)
     endif()
-    find_package(pmem 1.10 REQUIRED COMPONENTS ${pmem_COMPONENTS})
+    find_package(pmdk 1.8 REQUIRED COMPONENTS ${pmdk_COMPONENTS})
```

Comment:
If the required version is really 1.8, why https://github.com/ceph/ceph/pull/40493 insisted on 1.10?  Was it just a general desire to use the latest bits at the time?

Reply:
Updating to `v1.8` is a requirement to solve the bug caused by RWL fragmentation.  Upgrading to `v1.10` is more a requirement for @cyx1231st . Because crimson may use `libpmem2` in the future, and this library will be introduced in `v1.10`, so I upgrade `pmdk` to `v1.10` directly to avoid updating again in the future. But in fact, only `v1.8` is required now. 
Because of these differences, I want to separate the versions and dependencies required by different modules in the next PR‘s [commit](https://github.com/ceph/ceph/pull/46260/commits/19cfd033a578e3c6e29ebefc077820a057d7be7c). `libpmem` in bluestore has no version requirements. `RWL` requires libpmemobj to be after `v1.8`. `Crimson` may require `libpmem2` in the system. But in `build_pmdk()`, we provide a version that can meet all the modules.

Reply:
Thanks for the explanation!